docs/developer-docs/latest/developer-resources/cli/CLI.md

```diff
@@ -6,10 +6,13 @@ canonicalUrl: https://docs.strapi.io/developer-docs/latest/developer-resources/c
 
 # Command Line Interface (CLI)
 
-Strapi comes with a full featured Command Line Interface (CLI) which lets you scaffold and manage your project in seconds.
+Strapi comes with a full featured Command Line Interface (CLI) which lets you scaffold and manage your project in seconds. The CLI works with both the `yarn` and `npm` package managers, however, `yarn` generally provides a better user experience.
```

Comment:
Interactive prompts (like entering your key in `npm run strapi export` or credentials in `npm run strapi admin:create-user`) currently do not work with `npm run` and I don't know the timeline for fixing it.

@marcoautiero do we want to officially recommend `yarn` for the CLI?

Reply:
let's recommend it for now, but we will soon fix the issues with npm to make a coherent experience between npm and yarn